starlingx/inventory/v1/interfaceDataNetworks/results.go

```diff
@@ -50,11 +50,11 @@ type InterfaceDataNetwork struct {
 
 	// DataNetworkUUID defines the system assigned unique UUID of the associated
 	// data network.
-	DataNetworkUUID string `json:"network_uuid"`
+	DataNetworkUUID string `json:"datanetwork_uuid"`
```

Comment:
Is this ok to change json name?

Reply:
The "datanetwork_uuid" should be correct according to: https://docs.starlingx.io/api-ref/config/api-ref-sysinv-v1-config.html#interface-data-networks